app/models/manageiq/providers/vmware/infra_manager/inventory/parser.rb

```diff
@@ -170,6 +190,18 @@ def parse_host_system(object, kind, props)
     parse_host_system_lans(switches, props)
   end
 
+  def parse_license_manager(object, kind, props)
+    props[:licenses].each do |license|
+      {
+        :name        => license.name,
+        :license_key => license.licenseKey,
+        :edition_key => license.editionKey,
+        :total       => license.total,
+        :used        => license.used
```

Comment:
Super-minor, but do rbvmomi objects not have a `.to_h` method?  If it did, this method becomes kind of trivial (except for the key renaming maybe).  Perhaps that's something we want to get into rbvmomi if it doesn't exist?

Reply:
True, these methods are just accessors over an `@props` hash so we could hit that directly.